matlab/test/arrow/type/tTimeUnit.m

```diff
@@ -13,29 +13,39 @@
 % implied.  See the License for the specific language governing
 % permissions and limitations under the License.
 classdef tTimeUnit < matlab.unittest.TestCase
-% Test class for arrow.type.timeUnit
+% Tests for the arrow.type.TimeUnit enumeration class
```

Comment:
Could you move this above the license header?

Reply:
Fixed.